src/components/GameAbout/GameAbout.module.scss

```diff
@@ -25,3 +25,12 @@
     height: 16px;
   }
 }
+
+.gameTitle {
+  padding-bottom: var(--space-md);
+  h2 {
+    margin: 0;
+    color: var(--color-neutral-200);
+    font-size: var(--text-6xl);
+  }
```

Comment:
Is it possible for us to use a type from our typography system?

Reply:
I think I forgot to define this here, because when I put the mock in stories it used the same typography we already used: Poppins. 🤔

 I will fix this, thanks for the tip